data/committees.js

```diff
@@ -4,6 +4,7 @@ export default [
 	{
 		name: 'Studio',
 		class: 'studio',
+		slug: 'studio',
 		tagline: 'Creative Expression through Games and VR',
```

Comment:
For the `external_link` variable, thoughts on explicitly writing: `external_link: null`, so the fields are consistent for all committees?

I guess this really depends on if every committee wants an external site / will eventually create one

Reply:
I don't mind, especially if it makes things clearer! Will address tomorrow!